src/components/v5/common/Fields/Input/Input.tsx

```diff
@@ -131,7 +131,7 @@ const Input: FC<InputProps> = ({
             {isDecoratedError ? (
               <InputPills message={customErrorMessage} status="error" />
             ) : (
-              <FormError isFullSize alignment="left">
+              <FormError isFullSize alignment="left" allowLayoutShift={false}>
```

Comment:
should we maybe just add the classname override here? maybe there's no need for additional props if we can use CSS overrides for this

Reply:
I prefer the additonal prop here as it clearly states the meaning of the extra CSS, it is also less code as we dont have to go into every `FormError` component to set these classNames 
```
allowLayoutShift ? 'mt-1' : 'absolute',
```

Reply:
yeah fair enough, I thought about this too, it's a different mode of error :D i can't think of a better name than `allowLayoutShift` , so lgtm